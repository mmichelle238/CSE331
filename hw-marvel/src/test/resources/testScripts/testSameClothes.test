# Tests that if a path has the same source
# and destination, it wouldn't print anything.

LoadGraph clothes newClothes.tsv

FindPath clothes pants pants