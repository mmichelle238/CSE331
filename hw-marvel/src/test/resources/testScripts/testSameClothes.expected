# Tests that if a path has the same source
# and destination, it wouldn't print anything.

loaded graph clothes

path from pants to pants: